packages/screens/Launchpad/components/LaunchpadTeamandInvestment.tsx

```diff
@@ -37,13 +36,14 @@ export const LaunchpadTeamandCondition: React.FC = () => {
         </BrandText>
         <SpacerColumn size={2} />
 
-        <TextInputCustom<NewCollectionTeamandConditionFormValues>
-          rules={{ required: true }}
+        <TextInputLaunchpadTandIValues
+          required
```

Comment:
If every `TextInputLaunchpadTandIValues` components have `required` flag you can just call that directly in the component, and you can delete this props 👍

Reply:
for `TextInputLaunchpadTandIValues` not possible because one of them does not require a `required` flag, but for the other component, I fixed it.